client/src/pages/Login.js

```diff
@@ -6,33 +6,36 @@ import Button from "../components/Button/Button";
 const Wrapper = styled.div`
   display: flex;
   background-image: url(${bgImg});
+  background-repeat: no-repeat;
+  background-position: center;
   height: 100vh;
```

Comment:
```suggestion
  min-height: 100vh;
```
you could run into trouble when your content is bigger than your viewport